daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -468,7 +468,7 @@ class EngineTest
           let,
           lookupPackage
         )
-      (tx.transaction isReplayedBy rtx.transaction) shouldBe true
+      Tx.isReplayedBy(tx.transaction, rtx.transaction) shouldBe Right(())
```

Comment:
Did you know that I'm not a big fan of the infix syntax? This makes me so happy. 😺

Reply:
I don't have a specific opinion about this, but I'm actually curious about why changing. Is it just a preference or is there some other reasoning behind it?

Reply:
It is more a question of preference, but here are some reasons of the change:
- It is done in the same way as `Node.isReplayedBy`, function that `Transaction.isReplayedBy` subsumes now.  
- I can give a name to the left hand side transaction inside the body of `isReplayedBy`. 
- The code of the comparison is not so tide to the transaction itself, we could imagine implementing comparison more or less lenient. In fact, I am considering moving the code to the Engine package